app/features/navbar/components/HelpAction.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+import HelpIcon from '@atlaskit/icon/glyph/question-circle';
+import Droplist, { Item } from '@atlaskit/droplist';
+
+import React, { Component } from 'react';
+
+import config from '../../config';
+import { openExternalLink } from '../../utils';
+
+type State = {
+
+    /**
+     * Whether the droplist is open or not.
+     */
+    droplistOpen: boolean
+};
+
+/**
+ * Help Action for Navigation Bar.
+ */
+class HelpAction extends Component< *, State> {
+    /**
+     * Initializes a new {@code HelpAction} instance.
+     *
+     * @inheritdoc
+     */
+    constructor() {
+        super();
+
+        this.state = {
+            droplistOpen: false
+        };
+
+        this._droplistToggle = this._droplistToggle.bind(this);
+        this._openTermsPage = this._openTermsPage.bind(this);
+        this._openPrivacyPage = this._openPrivacyPage.bind(this);
+    }
+
+    _droplistToggle: (*) => void;
+
+    /**
+     * Toggles the droplist.
+     */
+    _droplistToggle() {
+        this.setState({
+            droplistOpen: !this.state.droplistOpen
+        });
+    }
+
+    _openTermsPage: (*) => void;
+
+    /**
+     * Opens Terms and Conditions Page in default browser.
+     */
+    _openTermsPage() {
+        openExternalLink(config.termsAndConditionsURL);
+    }
+
+    _openPrivacyPage: (*) => void;
+
+    /**
+     * Opens Privacy Policy Page in default browser.
+     */
+    _openPrivacyPage() {
+        openExternalLink(config.privacyPolicyURL);
+    }
+
+    _sendFeedback: (*) => void;
+
+    /**
+     * Opens Support/Feedback Email.
+     */
+    _sendFeedback() {
+        openExternalLink(config.feedbackURL);
```

Comment:
Missing the binding in the constructor. I know it's not technically needed now, but it will once we move config to redux and we need access to this.props here.

Reply:
Oh Yeah. My mistake.

Reply:
Updated.